inst/htmlwidgets/datatables.js

```diff
@@ -904,7 +904,7 @@ HTMLWidgets.widget({
         // server-side tables, we have to *real* row indices are in `selected1`
         if (server) table.on('draw.dt', selectRows);
         methods.selectRows = function(selected) {
-          selected1 = selected ? selected : [];
+          selected1 = selected ? $.isArray(selected) ? selected : [selected] : [];
```

Comment:
I think you can just use `$.makeArray()`, e.g., `selected1 = $.makeArray(selected)` (`undefined` / `null` will be converted to `[]`).

Reply:
Thanks for pointing out this. I'll take a note :laughing: